### ISSUE TITLE
Sharing directory with Windows/NTFS partition

### ISSUE THREAD

**Issue by CaptainJawZ** [abraunegg/onedrive#571]
Not a very important feature request but it would be useful if you could use your already existing files from a NTFS partition to sync everything without having to redownload/sync a bunch of files whenever you boot to Windows/Linux.

**Comment by abraunegg** [comment]
@CaptainJawZ 
This is more of 'how you use your Linux system' - it is possible, but you will need to investigate NTFS-3G and mount your partition / files accordingly. Windows encryption may get in the way.

Closing this as this is out of scope for 'onedrive'

**Comment by CaptainJawZ** [comment]
@abraunegg 
Fair enough! I'll just comment this in case someone needs it. Since I spent the entire day figuring out how to get it to work. 
I did managed to make it work, there are a few things to consider but overall.

1. Keep in mind this is experimental and puts your files at risk. 
2. Enable full sync on both Windows and ur onedrive linux configuration. I recommend this because if there is a folder that is not synching on either end it can cause problems on the server. 
3. Set your distro to auto/mount your Windows partition, either by the settings or fstab

Now onto the things to keep in mind:
1. First you configure your onedrive ~/.config/onedrive/config file, # comment down most stuff that has to do with skipping dirs or files. And uncomment sync_root_files = true, so all your files are synched. 
2. Add the folder containing your windows files to the config file
2. Wether you run the service or onedrive --synchronize NEVER add the --resync flag as this will re-download and overwrrite your files, this doesnt necesarely means data-loss but when you switch to windows those files will re-sync, and it will be an endless loop of uploading your files to the cloud. 

Now I dont know how recommended it is to work onedrive without a fully built sync database but I've been deleting/renaming/adding/removing/modifying files for several hours on several different directories and they all seem to sync just fine. 

I do not recommend to add a LOT of files while the onedrive service is not running as this can cause some duplicates. 

but yeah I seem to have got onedrive to work on a folder from a ntfs partition containing about 600gb of data without huge delays on the sync or having to --resync the database. 

And when I go to Windows 9 out of 10 times it doesnt resync the files I just modified as I suppose it can scan they are up to date. So it's almost a seamless transition.
